benchmark/process/coverage.js

```diff
@@ -0,0 +1,31 @@
+// This benchmark is meant to exercise a grab bag of code paths that would
+// be expected to run slower under coverage.
+'use strict';
+
+const common = require('../common.js');
+const bench = common.createBenchmark(main, {
+  n: [1e5]
+});
+const path = require('path');
+const { rmSync } = require('fs');
+const { spawnSync } = require('child_process');
+const tmpdir = require('../../test/common/tmpdir');
+
+const coverageDir = path.join(tmpdir.path, `./cov-${Date.now()}`);
+
+function main({ n }) {
+  bench.start();
+  const result = spawnSync(process.execPath, [
+    require.resolve('../fixtures/coverage-many-branches'),
+  ], {
+    env: {
+      NODE_V8_COVERAGE: coverageDir,
+      N: n,
+    }
```

Comment:
This should extend `env` rather than replace it, or at least it needs to preserve `LD_LIBRARY_PATH` for the shared library builds.

Reply:
@richardlau aha, I'd incorrectly assumed it was inspector session related, or something along those lines.